python/py-spiceypy/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
+
+github.setup	AndrewAnnex SpiceyPy v3.1.1 
+version	3.1.1	
+
+checksums           rmd160  eb48ae08ee964c659bb61a4bcfc8ce6db8169e74 \
+                    sha256  6281e703fd7c8c8a894c0693ac7b336f26b595c1a2c54666d8093d215d07211e \
+                    size    350722
+
+name             	py-spiceypy
+platforms        	darwin
+maintainers      	nomaintainer
+license          	MIT
+description	 	SpiceyPy: The NASA JPL NAIF SPICE toolkit wrapper for Python
+long_description	 ${description}
+
+python.versions         37 38
+
+universal_variant no
```

Comment:
please remove this line

Reply:
Which? You marked four lines.

Reply:
there is always some more code shown... the comment applies here to `universal_variant no`

Reply:
... I'm guessing the last one.